docs/client_configuration.rst

```diff
@@ -0,0 +1,79 @@
+Client Configuration
+====================
+
+`elasticsearch-lua`_ was designed with the aim to allow users to configure
+almost all of the parameters. The standard way of creating and configuring
+the client is:
+
+.. code-block:: lua
+
+  local client = elasticsearch.client{
+    hosts = {
+    -- array of client hosts
```

Comment:
`client hosts` seems weird to me. It should be something like elasticsearch hosts/nodes